.github/workflows/test.yml

```diff
@@ -16,19 +16,23 @@ jobs:
     strategy:
       matrix:
         node-version: [18.x, 22.x]
+        react-version: ['18.0.*', '18.1.*', '18.2.*', '18.3.*', '19.0.*']
```

Comment:
This can probably be simplified to `['18.*', '19.*']`, and test only against latest versions. It would also avoid having to remember to add future versions.